src/node_crypto.cc

```diff
@@ -3684,8 +3684,8 @@ void Hmac::HmacInit(const FunctionCallbackInfo<Value>& args) {
 bool Hmac::HmacUpdate(const char* data, int len) {
   if (!initialised_)
     return false;
-  HMAC_Update(&ctx_, reinterpret_cast<const unsigned char*>(data), len);
-  return true;
+  int r = HMAC_Update(&ctx_, reinterpret_cast<const unsigned char*>(data), len);
+  return r == 1;
```

Comment:
Wouldn't `!= 0` be a better conversion from truthiness to a bool?

HMAC_Update isn't actually documented as to its return value (meaning or value range), its just mentioned it used to return void. At least for SHA1, the call chain seems to still end up in a call of a function that returns `void`... not sure how that is supposed to work, I hope it doesn't mean the return value is random stack garbage, I'm doing a debug build to check